File: HW1/src/python/4.py
import pandas as pd
import numpy as np

test_url = "http://s3.amazonaws.com/assets.datacamp.com/course/Kaggle/test.csv"
test = pd.read_csv(test_url)

# Initialize a Survived column to 0
test["Survived"] = 0

# Set Survived to 1 if age is less than 18 and to 0 if age in greater equal than 18
test["Survived"][test["Age"] < 18] = 1
test["Survived"][test["Age"] >= 18] = 0

PassengerId =np.array(test["PassengerId"]).astype(int)
my_solution = pd.DataFrame(test[["Survived"]].as_matrix(), PassengerId, columns = ["Survived"])
my_solution.to_csv("my_solution_three.csv", index_label = ['PassengerId'])

File: HW1/src/other/4.R
# Import the training set: train
train_url <- "http://s3.amazonaws.com/assets.datacamp.com/course/Kaggle/train.csv"
train <- read.csv(train_url)

# Create the column child, and indicate whether child or no child
train$Child <- NA
train$Child[train$Age < 18] <- 1
train$Child[train$Age >= 18] <- 0

# Two-way comparison
prop.table(table(train$Child,train$Survived),1)

# Import the testing set: test
test_url <- "http://s3.amazonaws.com/assets.datacamp.com/course/Kaggle/test.csv"
test <- read.csv(test_url)

PassengerId <- as.vector(test$PassengerId)
test_one <- data.frame(PassengerId)
test_one$Survived <- 0
test_one$Survived[test$Age < 18] <- 1 
write.csv(test_one, file = "third_prediction.csv",row.names=FALSE)
